themes/base.theme.bash

```diff
@@ -8,9 +8,20 @@ SCM_THEME_PROMPT_DIRTY=' ✗'
 SCM_THEME_PROMPT_CLEAN=' ✓'
 SCM_THEME_PROMPT_PREFIX=' |'
 SCM_THEME_PROMPT_SUFFIX='|'
+SCM_THEME_BRANCH_PREFIX=''
+SCM_THEME_TAG_PREFIX='tag:'
+SCM_THEME_COMMIT_PREFIX='commit:'
+SCM_THEME_REMOTE_PREFIX=''
+
+SCM_GIT_SHOW_DETAILS=${SCM_GIT_SHOW_DETAILS:=false}
```

Comment:
I'm inclined to default this to `true` so things don't change for users who are currently expecting the prompt to contain git information.